src/components/pbicons/doit.sh

```
@@ -0,0 +1,41 @@
+map["xx09"]="CP2"
+map["xx10"]="D"
+map["xx11"]="DD"
+map["xx12"]="KD"
+map["xx13"]="P"
+map["xx14"]="PD"
+map["xx15"]="PP"
+map["xx16"]="T"
+map["xx17"]="CP-gbcp"
+map["xx18"]="CP-2gbcp"
+
+for file in ${!map[@]}; do
```

Shellcheck (0.10.0)

[error] Double quote array expansions to avoid re-splitting elements.

(SC2068)